client/src/components/App/Content/InvestigationForm/TabManagement/ExposuresAndFlights/ExposuresAndFlights.tsx

```diff
@@ -196,6 +195,7 @@ const ExposuresAndFlights : React.FC<Props> = ({ id }: Props): JSX.Element => {
           investigation: investigationId,
           user: userId
       });
+      setFormState(investigationId, id, true);
```

Comment:
why moving it to the then?

Reply:
setFormState is responsible for moving tabs and i want to move tab after saving the data